examples/rstudio/rstudio_auth

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# rstudio_auth method from https://github.com/nickjer/singularity-rstudio
+
+## Confirm username is supplied
+#if [[ $# -ne 1 ]]; then
+#  exit 1
+#fi
+#USERNAME="${1}"
+#
+## Confirm password environment variable exists (set in start_rstudio.sh)
+#if [[ -z ${RSTUDIO_PASSWORD} ]]; then
+#  exit 1
+#fi
+#
+## Read in the password from user
+#read -s -p "Password: " PASSWORD
+#echo ""
+#
+#if [[ ${USERNAME} == ${USER} && ${PASSWORD} == ${RSTUDIO_PASSWORD} ]]; then
+#  exit 0
+#else
+#  exit 1
+#fi
```

Comment:
Why is the whole script commented out?

Reply:
You had discussed previously just turning off authentication. I thought having access to how that might work could be beneficial to users, but it's probably better to just use bin/true and provide a link in a comment to the PAM helper script from the singularity example. https://github.com/nickjer/singularity-rstudio/blob/master/rstudio_auth.sh